Review.java

```diff
@@ -80,9 +84,83 @@ public static String textToString( String fileName )
       System.out.println("Unable to locate " + fileName);
     }
     //make sure to remove any additional space that may have been added at the end of the string.
+    //totalSentiment("SimpleReview.txt");
     return temp.trim();
+  }
+     /**
+   * Returns the word after removing any beginning or ending punctuation
+   */
+  public static String removePunctuation( String word )
+  {
+    while(word.length() > 0 && !Character.isAlphabetic(word.charAt(0)))
+    {
+      word = word.substring(1);
+    }
+    while(word.length() > 0 && !Character.isAlphabetic(word.charAt(word.length()-1)))
+    {
+      word = word.substring(0, word.length()-1);
+    }
+    
+    return word;
+  }
+
+  
+  public static double totalSentiment(String fileName)
+  {
+   //open file
+   //pick each word -> when I find a space, all the stuff before was a word. But I've got to remove punctuation.
+   //look up sentiment value
+   //add to total
+   //return value

```

Comment:
Good commentary/pseudocode. Easy to tell what is happening. Maybe place each line of pseudocode with the code segment it is describing to provide more clarity